composer.json

```diff
@@ -4,56 +4,58 @@
     "keywords": ["framework", "laravel"],
     "license": "MIT",
     "type": "project",
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/hotmeteor/Generators.git"
+        }
+    ],
     "require": {
-        "php": "^7.1.3",
-        "algolia/algoliasearch-client-php": "^1.27",
-        "appstract/laravel-opcache": "^2.0",
-        "aws/aws-sdk-php-laravel": "^3.1",
+        "php": "^8.0",
+        "algolia/algoliasearch-client-php": "^3.0",
+        "appstract/laravel-opcache": "^4.0",
+        "aws/aws-sdk-php-laravel": "^3.6",
         "bugsnag/bugsnag-laravel": "^2.0",
         "curl/curl": "^1.9",
-        "doctrine/dbal": "^2.5",
+        "doctrine/dbal": "^3.1",
         "fideloper/proxy": "^4.0",
-        "geoip2/geoip2": "^2.9",
+        "geoip2/geoip2": "^2.11",
         "google/recaptcha": "^1.1",
         "guzzlehttp/guzzle": "~6.0",
         "intervention/image": "^2.4",
-        "intouch/laravel-newrelic": "~2.0",
```

Comment:
I don't see a new version of laravel-newrelic added

Reply:
The package:  `In-Touch/laravel-newrelic` currently doesn't support laravel 8. I had removed it and I have replaced it by other package: `ratehub/laravel-newrelic`